### PR TITLE
[plugin.googledrive] 1.3.3

### PR DESCRIPTION
**Pull request by cguZZman**
### Description

* Fix playback
* Fix search
* Fix videos in Google Photos
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by cguZZman** [comment]
Failure is due to dependency on https://github.com/xbmc/repo-scripts/pull/1300 which should get resolved once it's merged

**Comment by cguZZman** [comment]
Had to push an updated version. Hope doesn't cost you any rework.

_Failure is due to dependency on xbmc/repo-scripts#1300 which should get resolved once it's merged_

**Comment by cguZZman** [comment]
Hope you guys have some time to merge this PR soon since the dependency was already merged and causing some compatibility issues. Not trying to be pushy here, just saying.

File: plugin.googledrive/addon.xml [hunk 1]
```diff
@@ -36,7 +36,20 @@ Play all your media from Google Drive including Videos, Music and Pictures (incl
 		<source>https://github.com/cguZZman/plugin.googledrive</source>
 		<forum>https://github.com/cguZZman/plugin.googledrive/issues</forum>
 		<website>https://addons.kodi.tv/show/plugin.googledrive</website>
+		<assets>
+			<icon>icon.png</icon>
+			<fanart>fanart.jpg</fanart>
+		</assets>
 		<news>
+v1.3.3 released Feb 14, 2020:
```
Comment (enen92) [inline plugin.googledrive/addon.xml]: please keep this  only for the last version in the next update.

Comment (cguZZman) [reply plugin.googledrive/addon.xml]: Noted. Will do.

**Comment by edevil** [comment]
Great news. When can we expect this plugin version to be seen by Kodi?

**Comment by cguZZman** [comment]
It's already available in Kodi's Repository

**Pull request merged**